README.md

```diff
@@ -522,6 +441,93 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Voilà](https://github.com/voila-dashboards/voila) ![](https://img.shields.io/github/stars/voila-dashboards/voila.svg?style=social) - Voilà turns Jupyter notebooks into standalone web applications that can e.g. be used as dashboards.
 
 
+##Industry-strength CV
```

Comment:
Still issue

Reply:
sorry, my bad; I remove the hyphen it works! @axsaucedo